src/timetable_gui.cpp

```diff
@@ -667,7 +728,7 @@ struct TimetableWindow : Window {
 			}
 
 			case WID_VT_CHANGE_TIME:
-				if (!_settings_client.gui.timetable_in_ticks) val *= Ticks::DAY_TICKS;
+				if (_settings_client.gui.timetable_mode != TimetableMode::Ticks) val *= TicksPerDayOrSecond();
```

Comment:
Special-case that could be removed if `TicksPerDayOrSecond()` also handled `TimetableMode::Ticks`.